week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,12 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  return "$" + ((price * 1.4) + (price * 1.4 * 0.01));
+}
+function convertToBRL(price) {
+ return "R$" + ((price * 4.8) + (price * 4.8 * 0.01));
```

Comment:
What if the 1% fee changes to 2% tomorrow? You have to modify it in two places. Can you think of a way to make sure both functions use the same fee always?